.travis.yml

```diff
@@ -43,7 +53,7 @@ jobs:
        - CXX=g++-8
        - ARGS="--with-flux-security --enable-sanitizer=address"
        - PRELOAD=/usr/lib/gcc/x86_64-linux-gnu/8/libasan.so
-       - ASAN_OPTIONS=detect_leaks=0 # TODO: fix/suppress tests so we can turn this on
+       - ASAN_OPTIONS=detect_leaks=0,start_deactivated=true # TODO: fix/suppress tests so we can turn this on
```

Comment:
Out of curiosity, what was the motivation for this flag?  Based on the docs, it looks like it delays allocating memory, but doesn't necessarily reduce memory footprint overall.  [In the docs](https://github.com/google/sanitizers/wiki/AddressSanitizerFlags), the line about this that stood out to me was "This is mainly intended to be used on Android.".

Reply:
@trws nicely described the reason in his commit message:
```
This causes asan to start in a low memory/low overhead mode until
instrumented code is loaded into the process.  In my environment this
lowers the asan run time for our full test suite by about 10%.
```

Reply:
Though looking at historical runtime, not sure it had an effect in the travis environment. :man_shrugging:

Reply:
Thanks @grondo!  Should've gone through the commits one-by-one. :man_facepalming: